docs/source/installation.mdx

```diff
@@ -9,19 +10,13 @@ import PodsRunScriptPanel from "../shared/pods-run-script-panel.mdx"
 import CarthageInstallationPanel from "../shared/carthage-installation-panel.mdx"
 import CarthageRunScriptPanel from "../shared/carthage-run-script-panel.mdx"
 
-Apollo iOS requires the latest Xcode, which can be installed from the [Mac App Store](http://appstore.com/mac/apple/xcode).
+Follow the steps below to add Apollo iOS to your app:
 
-Follow along with these steps (described in detail below) to use Apollo iOS in your app:
+## 1. Update Xcode
 
-1. Install the Apollo framework into your project and link it to your application target
-1. Add a schema file to your target directory
-1. (optional) Install the Xcode add-ons to get syntax highlighting for your `.graphql` files 
-1. Create `.graphql` files with your queries or mutations and add them to your target
-1. Add a code generation build step to your target
-1. Build your target
-1. Add the generated API file to your target
```

Comment:
Why remove the outline here?

Reply:
My reasoning was that:

* The outline is effectively represented by the right-side table of contents
* Seeing the outline straight away in the article body might cause the installation process to appear more intimidating than it actually is
* It helps people just sort of get going

I'm open to restoring it though!

Reply:
All fair reasons, I was mostly curious!